### PR TITLE
Update takeoperator.md

### PR DESCRIPTION
**Pull request by msftmatt**
Fix typo

### CODE REVIEW

**Comment by PRMerger16** [comment]
@msftmatt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/query/takeoperator.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ the source data is sorted.
 when browsing data interactively, but be aware that it doesn't guarantee any consistency
 in its results when executing multiple times, even if the data set hasn't changed.
 
-Even is the number of rows returned by the query is not explicitly limited
+Even if the number of rows returned by the query is not explicitly limited
```
Comment (orspod) [inline data-explorer/kusto/query/takeoperator.md:36]:
```suggestion
Even if the number of rows returned by the query isn't explicitly limited
```

**Comment by PRMerger8** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
@msftmatt Thanks so much for taking the time to leave us your feedback on MSDocs. I am approving your change.
Appreciate your contribution to our community.

**Pull request merged**